### PR TITLE
Configure markdown generation

### PR DESCRIPTION
**Pull request by henryre**
Run `tox -e markdown` to generate all markdown webpages. Just add the page you want to .web.yml. Will deploy via Travis in next PR.

**Test plan**
Testing locally

### CODE REVIEW

File: scripts/build.py [hunk 1]
```diff
@@ -75,6 +90,59 @@ def ipynb(self) -> str:
...
+            notebooks = get_notebooks(cfg["notebook"])
+            if len(notebooks) > 1:
+                raise ValueError(f"Multiple notebooks found in {cfg['notebook']}")
+            notebook = notebooks[0]
+        tutorial_webpages.append(
+            TutorialWebpage(
+                ipynb_path=notebook.ipynb,
+                title=cfg["title"],
+                description=cfg["description"],
+                order=i + 1,
```
Comment (paroma) [inline scripts/build.py]: is the default ordering however the files are read, unless set explicitly?

File: scripts/build.py [hunk 2]
```diff
@@ -75,6 +90,59 @@ def ipynb(self) -> str:
...
+        return HEADER_TEMPLATE.format(
+            title=self.title, description=self.description, order=self.order
+        )
+
+
+def parse_web_yml(tutorial_dir: Optional[str]) -> List[TutorialWebpage]:
+    with open(WEB_YML, "r") as f:
+        web_config = yaml.safe_load(f)
+    tutorial_webpages = []
+    for i, cfg in enumerate(web_config["tutorials"]):
```
Comment (paroma) [inline scripts/build.py]: is the ordering set alphabetically by default? where can the ordering be enforced (if via website, that works too)

File: .web.yml [hunk 3]
```diff
@@ -0,0 +1,25 @@
+tutorials:
```
Comment (paroma) [inline .web.yml]: is there a way to make sure there's an entry for each notebook?

File: .web.yml [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+    notebook: spam/01_spam_tutorial.ipynb
+  - title: Introduction to Transformation Functions
+    description: Data augmentation for spam classification
+    notebook: spam/02_spam_data_augmentation_tutorial.ipynb
+  - title: Introduction to Slicing Functions
+    description: Monitoring models for spam classification
+    notebook: spam/03_spam_data_slicing_tutorial.ipynb
+  - title: Hybrid Crowd Labeling Workflows in Snorkel
+    description: Programmatic and crowdworker labels for sentiment analysis
+    notebook: crowdsourcing
```
Comment (paroma) [inline .web.yml]: why are some entries for `notebook` are formatted as `*.ipynb` and others aren't?

File: scripts/build.py (hunk 2 above)
Comment (henryre) [reply scripts/build.py]: just by order in yaml

File: .web.yml (hunk 4 above)
Comment (henryre) [reply .web.yml]: if only one notebook in directory (most cases), find it automatically

File: .web.yml (hunk 3 above)
Comment (henryre) [reply .web.yml]: do we want that? example: no clear way to render drybell right now

Comment (henryre) [reply .web.yml]: there's just the one master webpage config here, so easy to add once if we notice something is missing for website

File: .web.yml (hunk 4 above)
Comment (paroma) [reply .web.yml]: nit but can we have the .ipynb everywhere so it's easier to pattern match

File: .web.yml (hunk 3 above)
Comment (paroma) [reply .web.yml]: if we're not requiring every tutorial to be rendered on the website, this is okay

**Pull request merged**